gui/build.py

```diff
@@ -79,6 +80,20 @@ def do_bundle():
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
+    # create a intaller
+    env = dict(os.environ)
+    p = re.search('D:\\\\Python27\\\\Lib', env['PYTHONPATH'])
```

Comment:
Hm, I remember you tell me on irc that I should get python path on users' system rather than path in virtual environment. So, I get my os environment path and pick python path from them. But env['PYTHONPATH'] give me a string contain three python path('D:\\Python27\\Lib;D:\\Python27\\bin;D:\\Python27\\Lib\\Script').
I though I could use regular expression to pick only one path from it, but the problem is not everyone install python on disk 'D' or maybe the python path is 'D:\xxxx\xxxx\Python27' and I don't know how to solve it.